web_client/src/components/ControlPanel.vue

```diff
@@ -204,92 +204,108 @@ export default {
-                <div
-                  rows="2"
-                  class="py-3"
-                  style="text-align: center; height: 70px"
-                >
-                  <div v-if="scanCachedPercentage < 1 && scanCachedPercentage > 0">
-                    <v-progress-circular
-                      :value="scanCachedPercentage * 100"
-                      color="blue"
-                    />
-                    <div> Loading... </div>
-                  </div>
-                </div>
-                <div
-                  v-if="lockOwner"
-                  class="grey--text d-flex"
-                  style="text-align: right; flex-direction:column; row-gap: 5px;"
-                >
-                  <span>{{ currentViewData.projectName }}</span>
-                  <div>
-                    <UserAvatar
-                      v-if="lockOwner"
-                      :target-user="lockOwner"
-                      as-editor
-                    />
-                    {{ currentViewData.experimentName }}
-                  </div>
-                </div>
-              </v-flex>
-              <v-textarea
-                v-model="currentViewData.experimentNote"
-                filled
-                :disabled="!experimentIsEditable"
-                no-resize
-                height="80px"
-                hide-details
-                class="mt-3"
-                name="input-experiment-notes"
-                label="Experiment Notes"
-                placeholder="There are no notes on this experiment."
-                @input="handleExperimentNoteChange"
-              />
-              <v-row no-gutters>
-                <v-col
-                  :class="newExperimentNote.length > 0 ? 'blue--text' : 'grey--text'"
-                  style="text-align: right"
-                  @click="handleExperimentNoteSave()"
-                >
-                  Save Note
-                </v-col>
-              </v-row>
-              <v-flex
-                class="d-flex ml-5"
-                style="flex-direction:column"
+                <v-progress-circular
+                  :value="scanCachedPercentage * 100"
+                  color="blue"
+                />
+                <div> Loading... </div>
+              </div>
+            </div>
+            <v-textarea
+              v-model="currentViewData.experimentNote"
+              filled
+              :disabled="!experimentIsEditable"
+              no-resize
+              height="60px"
+              hide-details
+              name="input-experiment-notes"
+              label="Experiment Notes"
+              placeholder="There are no notes on this experiment."
+              style="margin-top: 15px;"
+              class="mx-3"
+              @input="handleExperimentNoteChange"
+            />
+            <v-row no-gutters>
+              <v-col
+                :class="newExperimentNote.length > 0 ? 'blue--text' : 'grey--text'"
+                style="text-align: right"
```

Comment:
Isn't there a vuetify class for this as well?

Reply:
Changed in [f47f9d9](https://github.com/OpenImaging/miqa/pull/538/commits/f47f9d99cc6d01d74b68bc856401f459866cc609)